README.md

```
@@ -1,3 +1,134 @@
+    ```
+    [ERROR] 해당 위치에 이동할 말이 존재하지 않습니다.
+    ```
+    - 예외처리 - 장기말의 이동 규칙에 어긋난 경우
+
+    ```
+    [ERROR] 장기말의 이동 규칙을 확인해주세요.
+    ```
+
+    - 예외처리 - 장애물로 인해 이동이 불가능한 위치인 경우
+
+    ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)